code/_onclick/hud/hud.dm

```diff
@@ -113,179 +117,86 @@ var/datum/global_hud/global_hud = new()
 	var/obj/screen/r_hand_hud_object
 	var/obj/screen/l_hand_hud_object
 	var/obj/screen/action_intent
-	var/obj/screen/move_intent
+	var/obj/screen/zone_select
+	var/obj/screen/pull_icon
+	var/obj/screen/throw_icon

```

Comment:
these obj/screen var are no longer mob vars but are vars of the /datum/hud. There's no reason to have them as mob vars.